R/Widget_TimeSeries.R

```diff
@@ -41,6 +41,48 @@ Widget_TimeSeries <- function(
     siteSelectLabelValue <- paste0("Highlighted ", siteSelectLabelValue, ": ")
   }
 
+  # rename results to account for rpt_* table refactor
```

Comment:
Ideally the widgets will take snapshot outputs as inputs. it's not entirely clear how `results` here maps to the snapshot outputs. 

Need to take a look at this and think through how that will work in the new charting paradigm. Going to be closely related to how we handle the mapping between `assessment$lResults` and `snapshot$lData` as discussed above.